influxd.yaml

```diff
@@ -61,6 +61,54 @@ subpackages:
         - bash
         - yq
 
+  - name: ${{package.name}}-${{package.version}}-bitnami-compat
+    description: "compat package with bitnami/influxdb image"
+    pipeline:
+      - uses: bitnami/compat
+        with:
+          image: influxdb
+          version-path: 2/debian-12
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/opt/bitnami/influxd/bin
+          cp ${{targets.destdir}}/usr/bin/influxd ${{targets.subpkgdir}}/opt/bitnami/influxd/bin/influxd
+    test:
+      environment:
+        contents:
+          packages:
+            - curl
+      pipeline:
+        - name: version tests
+          runs: |
+            /opt/bitnami/influxd/bin/influxd version | grep ${{package.version}}
+        - name: Start the influxd server
+          uses: test/daemon-check-output
+          with:
+            start: |
+              /opt/bitnami/influxd/bin/influxd run
```

Comment:
We need to provide all required env vars here, example here: https://github.com/wolfi-dev/os/blob/45214d8b0fd5427711f02174c01a3bc1c175b4ce/repmgr-17.yaml#L184-L201